connectors/http/polling/element-templates/http-polling-connector.json

```diff
@@ -0,0 +1,491 @@
+        "equals": "basic"
+      }
+    },
+    {
+      "label": "Request body",
+      "description": "Payload to send with the request",
+      "group": "input",
+      "type": "Text",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.body"
+      },
+      "condition": {
+        "property": "method",
+        "oneOf": [
+          "post",
+          "put",
+          "patch",
+          "delete"
+        ]
+      },
+      "optional": true
+    },
+    {
+      "label": "Correlation key (process)",
+      "type": "String",
+      "group": "activation",
+      "feel": "required",
+      "description": "Sets up the correlation key from process variables",
+      "binding": {
+        "type": "bpmn:Message#zeebe:subscription#property",
+        "name": "correlationKey"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Correlation key (payload)",
+      "type": "String",
+      "group": "activation",
+      "feel": "required",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "correlationKeyExpression"
+      },
+      "description": "Extracts the correlation key from the incoming message payload",
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Condition",
+      "type": "String",
+      "group": "activation",
+      "feel": "required",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "activationCondition"
+      },
+      "description": "Condition under which the connector triggers. Leave empty to catch all events. <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/http-webhook/#make-your-http-webhook-connector-for-receiving-messages-executable' target='_blank'>See documentation</a>"
+    },
+    {
+      "label": "Timer",
+      "description": "A time duration for the lifecycle of polling connector defined as ISO 8601 durations format.<br> <li>PT15S for 15 seconds,</li><li>PT1H30M for 1 hour and 30 minutes,</li><li>P14D for 14 days.</li> <a href='https://docs.camunda.io/docs/components/modeler/bpmn/timer-events/#time-duration' target='_blank'>How to configure a time duration</a>",
+      "group": "timer",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.timer"
+      },
+      "optional": true,
+      "feel": "optional",
+      "constraints": {
+        "notEmpty": false
+      }
+    },
+    {
+      "label": "Message After Timer Ends",
+      "description": "Response if polling connector's time expires",
+      "group": "timer",
+      "type": "String",
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.timer.message"
+      },
+      "optional": true,
+      "feel": "optional",
+      "constraints": {
+        "notEmpty": false
+      }
+    },
+    {
+      "label": "Connection timeout",
```

Comment:
I would prefer to move this field into the `endpoint` group (to make it visually close to the HTTP request so users can easily understand the purpose of this field)